src/backend/marsha/urls.py

```diff
@@ -38,6 +38,8 @@
     path(
         "lti/documents/<uuid:uuid>", DocumentLTIView.as_view(), name="document_lti_view"
     ),
+    # Public resources
+    path("video/<uuid:uuid>", VideoLTIView.as_view(), name="video_public"),
     # API
```

Comment:
Did you forget to add documents?

Reply:
No I didn't, I wanted to discuss if wee add them or not.